batchflow/models/tf/encoder_decoder.py

```diff
@@ -251,9 +259,16 @@ def embedding(cls, inputs, name='embedding', **kwargs):
         -------
         tf.Tensor
         """
+        steps = kwargs.get('num_stages', 1)
```

Comment:
Having `step` and a list of embeddings seems like a total overkill. Let's choose only one of those or even reject both in favor of method overloading

Reply:
Looks like it is quite easy to replicate `num_stages` parameters via list of embeddings, so I've decided to remove it.